third_party/maya/lib/pxrUsdMayaGL/proxyDrawOverride.cpp

```diff
@@ -194,11 +262,418 @@ UsdMayaProxyDrawOverride::supportedDrawAPIs() const
 #endif
 }
 
+static void
+drawBox(GLfloat size, GLenum type)
+{
+	static const GLfloat n[6][3] =
+	{
+		{ -1.0, 0.0, 0.0 },
+		{ 0.0, 1.0, 0.0 },
+		{ 1.0, 0.0, 0.0 },
+		{ 0.0, -1.0, 0.0 },
+		{ 0.0, 0.0, 1.0 },
+		{ 0.0, 0.0, -1.0 }
+	};
+	static const GLint faces[6][4] =
+	{
+		{ 0, 1, 2, 3 },
+		{ 3, 2, 6, 7 },
+		{ 7, 6, 5, 4 },
+		{ 4, 5, 1, 0 },
+		{ 5, 6, 2, 1 },
+		{ 7, 4, 0, 3 }
+	};
+	GLfloat v[8][3];
+	GLint i;
+
+	v[0][0] = v[1][0] = v[2][0] = v[3][0] = -size / 2;
+	v[4][0] = v[5][0] = v[6][0] = v[7][0] = size / 2;
+	v[0][1] = v[1][1] = v[4][1] = v[5][1] = -size / 2;
+	v[2][1] = v[3][1] = v[6][1] = v[7][1] = size / 2;
+	v[0][2] = v[3][2] = v[4][2] = v[7][2] = -size / 2;
+	v[1][2] = v[2][2] = v[5][2] = v[6][2] = size / 2;
+
+	for (i = 5; i >= 0; i--) {
+		glBegin(type);
+		glNormal3fv(&n[i][0]);
+		glVertex3fv(&v[faces[i][0]][0]);
+		glVertex3fv(&v[faces[i][1]][0]);
+		glVertex3fv(&v[faces[i][2]][0]);
+		glVertex3fv(&v[faces[i][3]][0]);
+		glEnd();
+	}
+}
+
+void
+UsdMayaProxyDrawOverride::_RenderBounds(
+	const MBoundingBox &bounds,
+	const GfVec4f &wireframeColor,
+	const MMatrix& worldViewMat,
+	const MMatrix& projectionMat)
+{
+	glPushAttrib(GL_ENABLE_BIT | GL_CURRENT_BIT | GL_LIGHTING_BIT);
+	glDisable(GL_LIGHTING);
+	glMatrixMode(GL_PROJECTION);
+	glPushMatrix();
+	glLoadMatrixd(projectionMat.matrix[0]);
+	glMatrixMode(GL_MODELVIEW);
+	glPushMatrix();
+	glLoadMatrixd(worldViewMat.matrix[0]);
+
+	glColor4fv((float*)&wireframeColor);
+	glTranslated(bounds.center()[0],
+		bounds.center()[1],
+		bounds.center()[2]);
+	glScaled(bounds.width(),
+		bounds.height(),
+		bounds.depth());
+	//glutWireCube(1.0);
+	drawBox(1.0, GL_LINE_LOOP);
+	glMatrixMode(GL_PROJECTION);
+	glPopMatrix();
+	glMatrixMode(GL_MODELVIEW);
+	glPopMatrix();
+	glPopAttrib(); // GL_ENABLE_BIT | GL_CURRENT_BIT | GL_LIGHTING_BIT
+}
+
+void
+UsdMayaProxyDrawOverride::_SetupLighting(
```

Comment:
This method should probably also be removed in favor of another utility method in px_vp20:
https://github.com/PixarAnimationStudios/USD/blob/022f7ef54ff8ce15120f34e61e078c1daf471153/third_party/maya/lib/px_vp20/utils.h#L54